portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,49 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+	<head>
+		<meta charset="UTF-8">
+		<title>Yuyan's Portfolio</title>
+		<link rel="stylesheet" href="style.css">
+		<script src="script.js"></script>
+	</head>
+	<body>
+		<div class="nav">
+			<ul>
+				<li><a class="active" href="index.html">Home</a></li>
+				<li><a href="project.html">Projects</a></li>
+				<li><a href="experience.html">Experiences</a></li>
+			</ul>
+		</div>
+		<div class="profile">
+			<img src="/images/IMG_8092.jpg">
+			<h3>Yuyan Ke</h3>
+			
+		</div>
+		<div id="content">
+			<h2>About Me</h2>
+			<p>
+			Welcome to my portfolio!! I am a rising junior at the School of Engineering and Applied Scientists (SEAS) of Columbia University in the City of New York 
+			majoring in computer science with a minor in entrepreneurship. My interest in computer science lies in application development and system programming. 
```

Comment:
I've always said "systems programming". And: I think that it's cool that you're interested in systems programming because that's my specialization in CS.🙂